tests/ext/aws/test_consumer.py

```diff
@@ -7,83 +7,77 @@
 
 
 @pytest.mark.asyncio
-async def test_get_queue_url(mock_boto_client_sqs, mock_get_queue_url):
-    with mock_boto_client_sqs as mock_sqs:
+async def test_get_queue_url(mock_boto_session_sqs, boto_client_sqs):
+    with mock_boto_session_sqs as mock_sqs:
         consumer = Consumer('queue-name')
         queue_url = await consumer.get_queue_url()
         assert queue_url.startswith('https://')
         assert queue_url.endswith('queue-name')
 
         assert mock_sqs.called
         assert mock_sqs.called_once_with('sqs')
-        assert mock_get_queue_url.called
-        assert mock_get_queue_url.called_once_with(QueueName='queue-name')
+        assert boto_client_sqs.get_queue_url.called
```

Comment:
`aiomock` does not support `assert_called_once_with`, but there are other ways to test that